repos/rust-lang/annotate-snippets-rs.toml

```diff
@@ -0,0 +1,15 @@
+org = "rust-lang"
+name = "annotate-snippets-rs"
+description = "Library for snippet annotations"
+bots = []
+
+[access.teams]
+compiler = "write"
+devtools = "write"
+
+[access.individuals]
+Muscraft = "write"
```

Comment:
I am unsure of the difference between `maintain` and `write` and which would suit me

Reply:
`maintain` allows editing projects, wikis, discussions, probably not needed here (?).